### DISCUSSION TITLE
ClashMateForAndroid更新配置问题

### DISCUSSION THREAD

**Discussion by cheetahchen** [MetaCubeX/mihomo#1021] [Q&A]
配置里包含“proxy-providers:”，无论是手动还是自动更新配置，都无法拉取新的proxy配置到本地，同样的配置文件在ClashVerge上功能正常。请问是客户端设置问题还是客户端功能问题啊？

**Comment by makaspacex** [comment]
没遇到过 一般是服务器那边的问题或者网络问题吧 需要具体排查配置文件
